apm-agent-plugins/apm-vertx/apm-vertx3-plugin/src/test/java/co/elastic/apm/agent/vertx/v3/VertxServerTest.java

```diff
@@ -50,7 +50,7 @@ protected boolean useSSL() {
     }
 
     @Override
-    protected String expectedVertxVersion() {
-        return "3.6";
+    protected int getMajorVersion() {
+        return 3;
```

Comment:
so, we would not check for the minor version in this case?
just wondering if this may become a problem. But actually shouldn't be